FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/common/logging-stack/logging_stack.h

```diff
@@ -0,0 +1,108 @@
+/*
+ * Logging Stack V1.0.0
```

Comment:
All the other files in the Demo/ directory have "FreeRTOS Kernel V10.3.0" for the copyright. Why is this one different? Is the logging stack meant to be a library?

Reply:
Most of the files in the Demo directory has FreeRTOS Kernel V10.3.0 copyright. @aggarw13 and I were thinking to version it with logging stack header. I think logging-stack is a utility and it can be versioned in the  same way as other demo files. We will reconfirm the versioning and raise another PR to address if there is any change required. Thanks.